Formula/calyptia.rb

```diff
@@ -2,14 +2,14 @@
 # frozen_string_literal: true
 
 # This file was generated by GoReleaser. DO NOT EDIT.
-class Calyptia < Formula
+class CalyptiaAT320 < Formula
```

Comment:
Why is it called like that "CalyptiaAT320"?

Reply:
Yeah that was a copy-paste error I resolved